automation/TaskManager.py

```diff
@@ -536,6 +540,7 @@ def _issue_command(
                         "command_status": command_status,
                         "error": error_text,
                         "traceback": tb,
+                        "duration": time.time_ns() - t1,
```

Comment:
Reading this again I noticed, that we discussed saving the time in millisecond, so this needs to be
```suggestion
                        "duration": int((time.time_ns() - t1)/1000000),
```